pallets/snarcos/src/benchmarking/suite.rs

```diff
@@ -0,0 +1,51 @@
+use frame_benchmarking::{account, benchmarks, vec};
+use frame_support::{traits::Get, BoundedVec};
+use frame_system::RawOrigin;
+
+use crate::{
+    benchmarking::{linear_input, linear_proof, linear_vk, xor_input, xor_proof, xor_vk},
+    *,
+};
+
+const SEED: u32 = 41;
+
+benchmarks! {
+
+    store_key {
+        let caller = account("caller", 0, SEED);
+        let identifier = [0u8; 4];
+        let l in 1 .. T::MaximumVerificationKeyLength::get();
+        let key = vec![0u8; l as usize];
+    } : _(RawOrigin::Signed(caller), identifier, key.clone())
+
+    verify_xor {
```

Comment:
these two puppies below  are not really "benchmarked" but after the call I understand that the PR is just for building some benchmarks scaffolding. What I could suggest is pre-generating  keys of various sizes (say exponentially  increasing or in a fibonacci series) and running benchmarks for those as inputs?

Reply:
I agree, but I would suggest postponing it to the next benchmarking PR, when we have https://github.com/Cardinal-Cryptography/HouseSnark/pull/4 merged; then I will be able to benchmark against more relations, proof systems and key sizes. WDYT?